generator/geometry_holder.hpp

```diff
@@ -202,6 +194,7 @@ class GeometryHolder
     auto cp = m_header.GetGeometryCodingParams(i);
 
     // Optimization: Store first point once in header for outer linear features.
+    // @pastk: how does it help? why not store all points in the outer?
```

Comment:
Store the first point (usually encoded with a large number of bytes) once, instead of 4 times.

Reply:
Makes a lot of sense!
Why don't we store the last point in the header also?

Reply:
Last point in line is delta-encoded and usually 1 byte :)